packages/jsii-pacmak/lib/targets/dotnet.ts

```diff
@@ -5,10 +5,37 @@ import path = require('path');
 import xmlbuilder = require('xmlbuilder');
 import { IGenerator } from '../generator';
 import logging = require('../logging');
-import { Target, TargetOptions } from '../target';
+import { PackageInfo, Target, TargetOptions } from '../target';
 import { shell } from '../util';
 
 export default class Dotnet extends Target {
+    public static toPackageInfos(assm: spec.Assembly): { [language: string]: PackageInfo } {
+        const packageId = assm.targets!.dotnet!.packageId;
+        const version = assm.version;
+        const packageInfo: PackageInfo = {
+            repository: 'Nuget',
+            url: `https://www.nuget.org/packages/${packageId}/${version}`,
+            usage: {
+                dotnet: {
+                    language: 'console',
+                    code: `dotnet add package ${packageId} --version ${version}`
+                },
+                csproj: {
+                    language: 'xml',
+                    code: `<PackageReference Include="${packageId}" Version="${version}" />`
```

Comment:
For .NET Full Framework, `PackageReference` is only supported in VS2017 ([docs](https://docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference)). Suggest to add a `packages.config` section for .NET Full Framework on VS2015 and earlier.